### DISCUSSION TITLE
开发环境下修改加载资源的路径

### DISCUSSION THREAD

**Discussion by midsmr** [umijs/umi#11995] [Q&A] [answered]
我想在开发中从应用A（GO/JAVA）中嵌入应用B（UmiJS）
<img width="392" alt="截屏2023-12-24 22 23 59" src="https://github.com/umijs/umi/assets/61633355/b65a6d80-4f65-4148-8776-79c1ddbb674a">
如何使这些资源带有前缀，可以让我在应用A中代理请求这些资源

**Comment by fz6m** [comment, marked as answer]
你在对应的路由里，把 umi 构建的产物 `index.html` 文件读了之后内容统一返回给用户，附带 html mime type 响应头，比如 `/path/to/*` 都是响应 html 文件内容，之后把 umi 构建的其他产物作为静态文件提供，这一块自己想办法，比如看一下请求路径是否存在就响应对应的文件内容（附带 mime type）。

当然可能也有现成的工具可以让你托管整个产物静态资源到某个路径，通用需要将所有请求都 fallback 到 `index.html` 。

最简单的方法是用 Nginx 来托管静态资源，如果你是 docker 部署，最好通过专门的静态资源服务（如 Nginx）来提供前端资源，这样比较方便。

更多或更具体的部署方式请自行探索。
